libcontainer/configs/devices.go

```diff
@@ -3,15 +3,19 @@ package configs
 import "github.com/opencontainers/runc/libcontainer/devices"
 
 type (
+	// Device is deprecated: use libcontainer/devices.Device instead.
 	// Deprecated: use libcontainer/devices.Device
```

Comment:
Hmm. We are saying the same thing twice, in a slightly different way.

I'd rather not add new documentation for whatever is deprecated.

Reply:
Perhaps we should remove these stubs now; they were included in the v1.0.0-rc95 tag, so _possible_ consumers of this would've (ideally) seen the deprecation.